static/js/controllers/edit-contact.js

```diff
@@ -86,6 +92,10 @@ var libphonenumber = require('libphonenumber/utils'),
           };
           $scope.category = $scope.contact.type === 'person' ? 'person' : 'place';
           $scope.contactId = null;
+          if ($scope.category === 'place') {
+            // set the Primary Contact field for places to be 'blank'
+            $scope.contact.contact = $scope.parents[0];
```

Comment:
1. where would this get persisted to?
2. could we pull the `null` contact out as a constant?  And if so, where would we put it?